src/payment/strategies/affirm/affirm-payment-strategy.ts

```diff
@@ -57,7 +58,17 @@ export default class AffirmPaymentStrategy implements PaymentStrategy {
             throw new PaymentArgumentInvalidError(['payment.methodId']);
         }
 
-        return this._store.dispatch(this._orderActionCreator.submitOrder({ useStoreCredit }, options))
+        const requestOptions = {
+            ...options,
+            params: {
+                include: [
+                    'lineItems.physicalItems.categories',
```

Comment:
You can use `OrderIncludes.DigitalItemsCategories` and `OrderIncludes.PhysicalItemsCategories`